roles/robottelo/defaults/main.yml

```diff
@@ -3,5 +3,26 @@ robottelo_url: https://github.com/SatelliteQE/robottelo.git
 robottelo_directory: /root/robottelo
 robottelo_virtualenv_path: /root/robottelo/venv
 robottelo_version: master
-robottelo_test_arguments:
-  - tests/foreman/endtoend/test_api_endtoend.py
+robottelo_junit_file: results.xml
+robottelo_test_type: endtoend # one of [tier1|tier2|tier3|destructive|upgrade|endtoend]
+robottelo_test_endpoints: # [api|ui|cli]
+  - api
+robottelo_test_args: 
+  options: >
```

Comment:
We usually create separate variables for cases like this.

Reply:
Yes, I had it separated, I changed it in reaction to @ehelms's comment about hash mapping, though I might misunderstood what he asked for

Reply:
I was thinking more like:

```
robottelo_test_options:
  endtoend: 'not stubbed'
  default: "not stubbed and {{ robottelo_tyest_type }}"
```